### ISSUE TITLE
FE - Fix accessibility issues

### ISSUE THREAD

**Issue by nancruz** [Politica-Para-Todos/comentAR#49]
After running Axe tool to check for accessibility issues, some were found that should be fixed. 

![image](https://user-images.githubusercontent.com/1250918/65326155-b342d100-dba8-11e9-9231-58533877afdc.png)
